code/game/gamemodes/objective.dm

```diff
@@ -697,6 +732,10 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 	return target
 
 /datum/objective/destroy/update_explanation_text()
+	if(delayed_objective)
+		explanation_text = "Your objective is to either destroy or steal an Artificial Intelligence. You will receive further information in a few minutes."
```

Comment:
```suggestion
		explanation_text = "Your objective is to either destroy an Artificial Intelligence. You will receive further information in a few minutes."
```

Reply:
Also lmao this doesn't really hide it huh

Reply:
It's probably fine to lump it into "Assassinate" bucket. Or just state it outright, since you can't really rush destroying an AI anyway. (whatever is simpler)